### PR TITLE
feature: tagged resource limits

### PR DESCRIPTION
**Pull request by shwstppr**
https://github.com/apache/cloudstack/pull/8362

----
📚 Documentation preview 📚: https://cloudstack-documentation--379.org.readthedocs.build/en/379/

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Comment by vladimirpetrov** [comment]
I think we should also mention the new option for tagged template and the VM deployment wizard changes, @shwstppr.

**Comment by rohityadavcloud** [comment]
Is this ready for merging @shwstppr ?

**Comment by shwstppr** [comment]
@rohityadavcloud yes

File: source/adminguide/usage.rst [hunk 1]
```diff
@@ -263,6 +263,23 @@ max.project.secondary.storage (GB)  Maximum secondary storage space that can be
...
+- resource.limit.host.tags -  A comma-separated list of tags for host resource limits. It applies to resource types - User VM, CPU, Memory.
+- resource.limit.storage.tags - A comma-separated list of tags for storage resource limits. It applies to resource types - Volume, Primary storage.
+
+The limits for tagged resources are a subset of the overall limits and the maximum
+can be the value of the overall limit for the particular resource type.
+
+|accountlimits.png|
+
+The administrator can view used and available capacity of such tagged resource
+alongwith the overall capacities in the zone and cluster view in the UI.
```
Comment (sureshanaparti) [inline source/adminguide/usage.rst:279]:
```suggestion
along with the overall capacities in the zone and cluster view in the UI.
```

**Review by sureshanaparti — APPROVED** [review]
minor comment, doc lgtm

**Comment by shwstppr** [comment]
@blueorangutan docbuild

File: source/adminguide/usage.rst [hunk 2]
```diff
@@ -263,6 +263,23 @@ max.project.secondary.storage (GB)  Maximum secondary storage space that can be
                                     Default is 400.
 =================================== =================================================================
 
+The administrator can also set limits for specific tagged host and storage
+resources for an account or domain. Such tags must be specified in the following
+global settings:
+
+- resource.limit.host.tags -  A comma-separated list of tags for host resource limits. It applies to resource types - User VM, CPU, Memory.
+- resource.limit.storage.tags - A comma-separated list of tags for storage resource limits. It applies to resource types - Volume, Primary storage.
```
Comment (vishesh92) [inline source/adminguide/usage.rst:271]:
```suggestion
- `resource.limit.host.tags` -  A comma-separated list of tags for host resource limits. It applies to resource types - User VM, CPU, Memory.
- `resource.limit.storage.tags` - A comma-separated list of tags for storage resource limits. It applies to resource types - Volume, Primary storage.
```

Comment (vishesh92) [reply source/adminguide/usage.rst:271]: This makes the documentation look better.

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Review by rohityadavcloud — APPROVED** [review]

**Pull request merged**